### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by eugrdn** [eugrdn/RSREU-training_Integration#4]
Attach more detailed info about lab: 
* check new API
* request params
* additional services
* tasks for the students

**Comment by sweatC** [comment]
Hi!
I think it may be useful:
1) Rewrite this JS using es6 modules(import/export).
2) Add some info about Promises(someone don`t know that code below return Promise).
```
MongoClient.connect(url)
      .then(db => (_db = db))
      .catch(err => Promise.reject(err));
```
3) I didn`t check new api(if you mean mongodb).

_P.S. Could you show previous labs?_

**Comment by eugrdn** [comment]
Hi @sweatC, thank for your attention. But no assistance is required here.

**Comment by eugrdn** [comment]
Closed by #5
